bc_obps/registration/api/v2/_user_operators/_current/operator.py

```diff
@@ -25,6 +25,7 @@
     tags=V2,
     description="""Retrieves data about the current user-operator and their associated operator.
     Declined user-operators are excluded from the results.""",
+    exclude_none=True,  # To exclude None values from the response (used for parent and partner arrays)
```

Comment:
Oh cool, I just learned that this is a Django feature. Is this to stop null values from causing erroneous validation errors?

Reply:
Yes, to avoid validation and display errors. If there's a None value, then the FE receives whatever default value we set in the ninja schema for the field. So we get stuff like parent_operators_array = []/None, which overrides our default jsonSchema values.